src/common/utils.ts

```diff
@@ -314,7 +314,7 @@ export const gucci = async (
   tmpl: string | unknown,
   args: { [key: string]: any },
   opts?: GucciOptions,
-): Promise<string | Record<string, unknown>> => {
+): Promise<string | Record<string, any>> => {
```

Comment:
This function returns a string or an object? Is that desirable?

Reply:
The return type depends on `opts` argument.